### ISSUE TITLE
Add ember-get-config to addon dependencies

### ISSUE THREAD

**Issue by gandalfar** [oskarrough/ember-wordpress#28]
This addon uses `ember-get-config` that doesn't get installed by default.

If this sounds correct, I can try doing a PR for it.

**Comment by oskarrough** [comment]
Hey, thank you but I'm not sure we're using that. Where would it be?

**Comment by gandalfar** [comment]
Here it's where it's imported

https://github.com/oskarrough/ember-wordpress/blob/21bf1a6ff308266e3169f3ba5d40bf3d870b3626/addon/adapters/wordpress.js#L2

**Comment by oskarrough** [comment]
Thank you. It is included in the dev dependencies here https://github.com/oskarrough/ember-wordpress/blob/master/package.json#L40. Does it not get installed?

**Comment by gandalfar** [comment]
Not for me:

```
➜  build ember new wp-test
installing app
  create .bowerrc
  create .editorconfig
  create .ember-cli
  create .jshintrc
  create .travis.yml
  create .watchmanconfig
  create README.md
  create app/app.js
  create app/components/.gitkeep
  create app/controllers/.gitkeep
  create app/helpers/.gitkeep
  create app/index.html
  create app/models/.gitkeep
  create app/resolver.js
  create app/router.js
  create app/routes/.gitkeep
  create app/styles/app.css
  create app/templates/components/.gitkeep
  create bower.json
  create config/environment.js
  create ember-cli-build.js
  create .gitignore
  create package.json
  create public/crossdomain.xml
  create public/robots.txt
  create testem.js
  create tests/.jshintrc
  create tests/helpers/destroy-app.js
  create tests/helpers/module-for-acceptance.js
  create tests/helpers/resolver.js
  create tests/helpers/start-app.js
  create tests/index.html
  create tests/integration/.gitkeep
  create tests/test-helper.js
  create tests/unit/.gitkeep
  create vendor/.gitkeep
Successfully initialized git.
Installed packages for tooling via npm.
Installed browser packages via Bower.
➜  build ember isst
➜  build cd Remodal/
➜  build cd ember-test
cd: no such file or directory: ember-test
➜  build cd ember 
➜  build cd wp-test
➜  wp-test git:(master) ember install https://github.com/oskarrough/ember-wordpress.git
Installed packages for tooling via npm.
WARNING: Could not figure out blueprint name from: "https://github.com/oskarrough/ember-wordpress.git". Please install the addon blueprint via "ember generate <addon-name>" if necessary.
Installed addon package.
➜  wp-test git:(master) ✗ ember v
ember-cli: 2.10.0
node: 6.9.2
os: darwin x64
```

`routes/test.js`
```
import Ember from 'ember';

export default Ember.Route.extend({
  model(params) {
    return this.store.query('post', {filter: {'name': 'test'}}).then(models => models.get('firstObject'));
  }
});
```

and then error in chrome:

![screenshot 2016-12-20 11 49 43](https://cloud.githubusercontent.com/assets/13806/21347786/6a03fbce-c6aa-11e6-994b-a9141221eda2.png)

**Comment by oskarrough** [comment]
What if you run `ember generate ember-wordress`?

**Comment by gandalfar** [comment]
hmm, blueprints don't work for me because I installed from github (`Unknown blueprint: ember-wordress`). That's probably unrelated, maybe I can try from npm.

**Comment by gandalfar** [comment]
Installing via npm (0.1.1) version doesn't provide me with blueprint (`Unknown blueprint: ember-wordress`)

**Comment by oskarrough** [comment]
@gandalfar I've updated ember-get-config to 0.2.0 on the master branch. If possible, please let me know if that works.

**Comment by gandalfar** [comment]
Same problem:
```
ember.debug.js:27883 Error while processing route: test Could not find module `ember-get-config` imported from `ember-wordpress/adapters/wordpress` Error: Could not find module `ember-get-config` imported from `ember-wordpress/adapters/wordpress`
    at missingModule (http://localhost:4200/assets/vendor.js:248:11)
    at findModule (http://localhost:4200/assets/vendor.js:259:17)
    at Module.findDeps (http://localhost:4200/assets/vendor.js:199:24)
    at findModule (http://localhost:4200/assets/vendor.js:262:11)
    at Module.findDeps (http://localhost:4200/assets/vendor.js:199:24)
    at findModule (http://localhost:4200/assets/vendor.js:262:11)
    at requireModule (http://localhost:4200/assets/vendor.js:29:15)
    at Class._extractDefaultExport (http://localhost:4200/assets/vendor.js:92529:20)
    at Class.resolveOther (http://localhost:4200/assets/vendor.js:92226:32)
    at Class.superWrapper [as resolveOther] (http://localhost:4200/assets/vendor.js:50290:22)
logError @ ember.debug.js:27883
error @ ember.debug.js:27824
triggerEvent @ ember.debug.js:27981
trigger @ ember.debug.js:55676
trigger @ ember.debug.js:56218
(anonymous) @ ember.debug.js:56026
tryCatch @ ember.debug.js:58287
invokeCallback @ ember.debug.js:58302
publish @ ember.debug.js:58270
publishRejection @ ember.debug.js:58205
(anonymous) @ ember.debug.js:37727
invoke @ ember.debug.js:336
flush @ ember.debug.js:404
flush @ ember.debug.js:528
end @ ember.debug.js:598
run @ ember.debug.js:721
join @ ember.debug.js:743
run.join @ ember.debug.js:21614
(anonymous) @ ember.debug.js:21677
mightThrow @ jquery.js:3570
process @ jquery.js:3638
ember.debug.js:16955 Error: Could not find module `ember-get-config` imported from `ember-wordpress/adapters/wordpress`
    at missingModule (loader.js:239)
    at findModule (loader.js:250)
    at Module.findDeps (loader.js:190)
    at findModule (loader.js:253)
    at Module.findDeps (loader.js:190)
    at findModule (loader.js:253)
    at requireModule (loader.js:20)
    at Class._extractDefaultExport (resolver.js:385)
    at Class.resolveOther (resolver.js:82)
    at Class.superWrapper [as resolveOther] (ember.debug.js:39737)
```

```
➜  wp-test git:(master) ✗ npm install
wp-test@0.0.0 /Users/gandalf/build/wp-test
└── ember-wordpress@0.1.2  (git+https://github.com/oskarrough/ember-wordpress.git#effe7d73fdb03f29ca402e4b6d03b37a0d62351e)
```

**Comment by gandalfar** [comment]
I'm on beta of `ember-cli`, if that matters:

➜  wp-test git:(master) ✗ ember version --verbose && npm --version
ember-cli: 2.11.0-beta.1
http_parser: 2.7.0
node: 6.6.0
v8: 5.1.281.83
uv: 1.9.1
zlib: 1.2.8
ares: 1.10.1-DEV
icu: 57.1
modules: 48
openssl: 1.0.2h
os: darwin x64
3.10.7

**Comment by oskarrough** [comment]
@null-null-null do you have any idea what it could be? Thank you.

**Comment by null-null-null** [comment]
Hey @oskarrough! I believe you'll need `ember-get-config` to be an actual `dependency`, rather than a `devDependency`. Let me know if that solves the issue. If so, I'll definitely need to update the `ember-get-config` docs to reflect that. Sorry about the confusion.

**Comment by oskarrough** [comment]
@null-null-null thank you.

@gandalfar sorry, could you test out master once again to verify? I've now moved it to dependencies.

**Comment by gandalfar** [comment]
Yes, this works now!

**Comment by oskarrough** [comment]
thanks for the help @null-null-null and @gandalfar!

https://github.com/oskarrough/ember-wordpress/releases/tag/0.1.3

**Comment by josephbergdoll** [comment]
Hi @oskarrough  — this may be related: I'm getting the following when I try to run `ember server`

```
DEPRECATION: Addon files were detected in `/Users/jb/Dropbox/Sites/Mast-2017/web/frontend/node_modules/ember-wordpress/addon`, but no JavaScript preprocessors were found for `ember-wordpress`. Please make sure to add a preprocessor (most likely `ember-cli-babel`) to in `dependencies` (NOT `devDependencies`) in `ember-wordpress`'s `package.json`.
```

**Comment by oskarrough** [comment]
Hey @josephbergdoll, thanks. I've moved it to `dependencies` and pushed it to master here.

Could you help me check if it works for you? Then I can push a new release.

**Comment by oskarrough** [comment]
Released 0.1.4 which seems to fix this. Do report if there's anything.

https://github.com/oskarrough/ember-wordpress/commit/d71c66915061deeced0bbece1a8effacef44adc8
